tests/acceptance/features/apiSharingNg/updateShareInvitations.feature

```diff
@@ -83,6 +83,75 @@ Feature: Update permission of a share
     """
 
 
+  Scenario: user removes expiration date of a share
```

Comment:
issue tags will be helpful to check the coverage and know the related issues with the scenario